assets/sass/modules/_qtip.scss

```diff
@@ -36,3 +36,50 @@
     }
   }
 }
+
+/*! Custom qtip styles, based on dark theme */
+.qtip-custom {
```

Comment:
aha, I think it's because `okta-signin-widget/.scss-lint.yml` disabled many rules..